.github/CHANGELOG.md

```diff
@@ -48,6 +48,9 @@
 * Add a Catalyst-specific wrapping class for Lightning Kokkos.
   [(#770)](https://github.com/PennyLaneAI/pennylane-lightning/pull/770)
 
+* Shot batching is made more efficient by executing all the shots in one go on LightningQubit.
```

Comment:
Nitpick, but I think we tend to add newer PR at the top of a section. I don't know whether it's a rule, and we could change the rule, but do we want a rule?

Reply:
That's okay! Let's add newer PR at the top of the section. I just added the reference yesterday :)